Lib/idlelib/autocomplete_w.py

```diff
@@ -242,37 +242,48 @@ def winconfig_event(self, event):
         self.is_configuring = True
         if not self.is_active():
             return
```

Comment:
I believe it is a bug to leave self.is_configuring True when returning.   It seems that `not self.active` should be added to the condition above.   I also remember doing some test that indicated that this function may still be executed past here more than once.  But another issue.

Reply:
True, though unrelated to the bug. Want to fix this in this PR while we're here?

Reply:
Tempted but no.  Partly discipline -- we have coherent problem and solution we agree on, lets merge it (when you are ready).  We have done really great today with good focus.  Partly I want to retry the test I remember to check for any anomaly.  Start with just a PR, if you want, *after* this is merged into you local clone.  Or start with wider issue Clean up autocomplete_w code.  There is a too-big function that I would like to split.  And maybe other things.  Oh, and maybe some tests needing review.